.slackbot.yml

```diff
@@ -0,0 +1,15 @@
+releaseSchedule:
+  ggetz: 2/1/2019
```

Comment:
@OmarShehata what account do these names correlate to?

Reply:
These are the team's Slack usernames.